Include/internal/pycore_typeobject.h

```diff
@@ -74,7 +79,7 @@ struct types_state {
 extern PyStatus _PyTypes_InitTypes(PyInterpreterState *);
 extern void _PyTypes_FiniTypes(PyInterpreterState *);
 extern void _PyTypes_Fini(PyInterpreterState *);
-
+void _PyTypes_AfterFork(void);
```

Comment:
```suggestion
extern void _PyTypes_AfterFork(void);
```

I don't think it matters. Just for consistency.